tests/models/registry.py

```diff
@@ -115,6 +115,9 @@ class _HfExamplesInfo:
     "StableLmForCausalLM": _HfExamplesInfo("stabilityai/stablelm-3b-4e1t"),
     "Starcoder2ForCausalLM": _HfExamplesInfo("bigcode/starcoder2-3b"),
     "SolarForCausalLM": _HfExamplesInfo("upstage/solar-pro-preview-instruct"),
+    "TeleChat2ForCausalLM": _HfExamplesInfo("Tele-AI/TeleChat2-115B",
```

Comment:
```suggestion
    "TeleChat2ForCausalLM": _HfExamplesInfo("Tele-AI/TeleChat2-115B",
```

To ease testing, please use a smaller model here.

Reply:
@shunxing12345 Seems that `TeleChat2-3B` is only released on modelscope. Can you upload the `TeleChat2-3B` to HF for test?

Reply:
@Isotr0py Hi, I've uploaded it to HF.  https://huggingface.co/Tele-AI/TeleChat2-3B